src/test/java/teammates/sqlui/webapi/GetStudentActionTest.java

```diff
@@ -0,0 +1,288 @@
+package teammates.sqlui.webapi;
+
+import static org.mockito.Mockito.when;
+
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.InstructorPermissionRole;
+import teammates.common.datatransfer.InstructorPrivileges;
+import teammates.common.util.Const;
+import teammates.storage.sqlentity.Account;
+import teammates.storage.sqlentity.Course;
+import teammates.storage.sqlentity.Instructor;
+import teammates.storage.sqlentity.Student;
+import teammates.storage.sqlentity.Team;
+import teammates.ui.output.StudentData;
+import teammates.ui.webapi.GetStudentAction;
+
+/**
+ * SUT: {@link GetStudentAction}.
+ */
+public class GetStudentActionTest extends BaseActionTest<GetStudentAction> {
```

Comment:
nit: please add some blank lines for better readability in some of the test methods